cfgov/v1/atomic_elements/molecules.py

```diff
@@ -246,6 +246,25 @@ class Meta:
         label = 'Phone'
 
 
+class ImageInset(blocks.StructBlock):
+    heading = blocks.CharBlock(required=False)
+    paragraph = blocks.RichTextBlock(required=False)
+    image = atoms.ImageBasic()
+    is_image_left = blocks.BooleanBlock(required=False,
+                                        label='Image float left')
+    image_width = blocks.ChoiceBlock(choices=ref.inset_image_widths,
```

Comment:
So I've deleted the changes in ref.py and just added the choices directly to the class. These values shouldn't really be changing so I don't think it warranted being added to ref.py.